index.md

```diff
@@ -83,37 +62,58 @@ We weight grades as follows:
 | Final Project | 40% | 25% |  
 | 4th Credit Project | | 15% |  
 
-You can expect the cutoffs to be approximately as follows
-98 - 100: A+  
-93 - 98: A
-90 - 93: A-
-87 - 90: B+
-83 - 87: B
-80 - 83: B-
-77 - 80: C+
-73 - 77: C
-70 - 73: C-
-60 - 70: D
-Below 60: F
+You can expect the cutoffs to be approximately as follows: 
```

Comment:
Are these necessary/do they provide any new information/are we going to stick by these? I'm in favor of dropping these cutoffs entirely.

Reply:
The A range doesn't show up for me
And I agree that they are not necessary and they also take up a lot of space
![Screenshot 2019-03-14 23 10 17](https://user-images.githubusercontent.com/14083891/54408846-a69d6780-46b1-11e9-983b-c4a7a28c4933.png)

Reply:
Removed